ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.20__multi-nightly.yaml

```diff
@@ -1417,6 +1417,125 @@ tests:
+  steps:
+    cluster_profile: azure-qe
+    env:
+      AZURE_COMPUTE_MULTIDISK_MOUNT_PATH: /var/lib/containers
+      AZURE_COMPUTE_MULTIDISK_TYPE: user-defined
+      AZURE_CONTROL_PLANE_MULTIDISK_TYPE: etcd
+      AZURE_INSTALL_USE_MINIMAL_PERMISSIONS: "yes"
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: Standard_D4ps_v5
+      FEATURE_GATES: '["AzureMultiDisk=true", "MultiDiskSetup=true"]'
+      FEATURE_SET: CustomNoUpgrade
+      OCP_ARCH: arm64
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-azure-ipi-multidisk
+- as: azure-ipi-multi-disk-swap-mini-perm-tp-amd-f28-destructive
+  capabilities:
+  - intranet
+  cron: 3 15 22 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      AZURE_COMPUTE_MULTIDISK_TYPE: swap
+      AZURE_CONTROL_PLANE_MULTIDISK_TYPE: swap
+      AZURE_INSTALL_USE_MINIMAL_PERMISSIONS: "yes"
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      FEATURE_GATES: '["AzureMultiDisk=true'', ''MultiDiskSetup=true", "NodeSwap=true"]'
+      FEATURE_SET: CustomNoUpgrade
+    test:
+    - chain: openshift-e2e-test-qe-destructive
+    workflow: cucushift-installer-rehearse-azure-ipi-multidisk
+- as: azure-ipi-multi-disk-swap-mini-perm-tp-arm-f14
+  capabilities:
+  - intranet
+  cron: 41 10 3,17 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      AZURE_COMPUTE_MULTIDISK_TYPE: swap
+      AZURE_CONTROL_PLANE_MULTIDISK_TYPE: swap
+      AZURE_INSTALL_USE_MINIMAL_PERMISSIONS: "yes"
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: Standard_D4ps_v5
+      FEATURE_GATES: '["AzureMultiDisk=true", "MultiDiskSetup=true", "NodeSwap=true"]'
+      FEATURE_SET: CustomNoUpgrade
+      OCP_ARCH: arm64
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-azure-ipi-multidisk
+- as: azure-ipi-multi-disk-user-defined-tp-amd-f28-destructive
+  capabilities:
+  - intranet
+  cron: 3 19 6 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      AZURE_COMPUTE_MULTIDISK_CATCHING_TYPE: ReadWrite
+      AZURE_COMPUTE_MULTIDISK_LUN_ID: "20"
+      AZURE_COMPUTE_MULTIDISK_MOUNT_PATH: /var
+      AZURE_COMPUTE_MULTIDISK_STORAGE_ACCOUNT_TYPE: Standard_LRS
+      AZURE_COMPUTE_MULTIDISK_TYPE: user-defined
+      AZURE_CONTROL_PLANE_MULTIDISK_CATCHING_TYPE: None
+      AZURE_CONTROL_PLANE_MULTIDISK_LUN_ID: "16"
+      AZURE_CONTROL_PLANE_MULTIDISK_MOUNT_PATH: /var/lib/containers
+      AZURE_CONTROL_PLANE_MULTIDISK_STORAGE_ACCOUNT_TYPE: StandardSSD_LRS
+      AZURE_CONTROL_PLANE_MULTIDISK_TYPE: user-defined
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      FEATURE_GATES: '["AzureMultiDisk=true", "MultiDiskSetup=true"]'
+      FEATURE_SET: CustomNoUpgrade
+    test:
+    - chain: openshift-e2e-test-qe-destructive
+    workflow: cucushift-installer-rehearse-azure-ipi-multidisk
+- as: azure-ipi-multi-disk-user-defined-mini-perm-tp-arm-f14
+  capabilities:
+  - intranet
```

Comment:
Since https://github.com/openshift/release/pull/67488, we do not need the `capabilities` setting any more.

Reply:
acked, I copied from other existing job configuration, do I need to remove them in this PR?

Reply:
Yeah, please. If I remember upstream already remove them in a batch.

Reply:
removed